### PR TITLE
Added the warning cone as a headpiece for Ian

### PR DESCRIPTION
**Pull request by denghis1**
## What Does This PR Do
This PR allows Ian to wear the warning cone as a headpiece. 
The cone was added to dog_fashion and a new sprite was added to corgi_head (thanks to mcramon! for the sprite!)

## Why It's Good For The Game
More fluff for the stations beloved pet, this is also easier to get a hold of after the craftable cone pr got merged.
Besides, who can say no to Ian and the omnicone himself?

## Images of changes

image

image

2j4PfX7YTd

## Changelog
:cl: denghis
tweak: Ian can now wear the cone on his head

/:cl:

### CODE REVIEW

**Review by Kyep — APPROVED** [review]

**Comment by Dashsora** [comment]
Yessssssss amazing work 👍

**Comment by Hyper721** [comment]
If this doesn't get added, I'm killing the entire staff team.

File: code/datums/dog_fashion.dm [hunk 1]
```diff
@@ -191,6 +191,10 @@
 	..()
 	desc = "That's Definitely Not [M.real_name]."
 
+/datum/dog_fashion/head/cone
+	name = "REAL_NAME of the cone"
```
Comment (AffectedArc07) [inline code/datums/dog_fashion.dm:195]: I feel this this leaves something trailing and missing.

`Ian of the cone` what?

**Comment by procdrone** [comment]
Change the name to "Safety officer Ian"

**Review by farie82 — APPROVED** [review]
Code itself looks good.

File: code/datums/dog_fashion.dm (hunk 1 above)
Comment (denghis1) [reply code/datums/dog_fashion.dm:195]: Would it be better if i just left it as "REAL_NAME" ?

Comment (AffectedArc07) [reply code/datums/dog_fashion.dm:195]: Yes

**Pull request merged**